store/displayPreferences.ts

```diff
@@ -0,0 +1,304 @@
+import { DisplayPreferencesDto } from '@jellyfin/client-axios';
+import { MutationTree, ActionTree, GetterTree } from 'vuex';
+import nuxtConfig from '~/nuxt.config';
+
+const stringToBoolean = (value: string) => value === 'True';
+const booleanToString = (value: boolean) => (value ? 'True' : 'False');
+
+export interface DisplayPreferencesState extends DisplayPreferencesDto {
+  CustomPrefs: {
+    [key: string]: string;
+  };
+}
+
+const defaultCustomPrefs: {
+  [key: string]: string;
+} = {
+  darkMode: booleanToString(
+    nuxtConfig.vuetify?.theme?.default === 'dark' || false
+  ),
+  locale: nuxtConfig.i18n?.defaultLocale || 'en'
+};
+
+const defaultState = (): DisplayPreferencesState => ({
+  CustomPrefs: defaultCustomPrefs
+});
+
+/**
+ * Methods to apply for each CustomPrefs property.
+ * Those callbacks are usually done on global variables such as $vuetify or $i18n,
+ * and those may not be present early in the application lifecycle.
+ * The try-catch are there to silence those errors.
+ */
+const updateMethods: { [key: string]: (value: string) => void } = {
+  darkMode: (value: string) => {
+    try {
+      window.$nuxt.$vuetify.theme.dark = stringToBoolean(value);
+    } catch (error) {
+      // eslint-disable-next-line no-console
+      console.error(error);
+    }
```

Comment:
```suggestion
    if (window.$nuxt) {
      window.$nuxt.$vuetify.theme.dark = stringToBoolean(value);
    }
```

Something like this should fix the issue.

Reply:
Done!